src/Foundation/Commerce/code/Infrastructure/Pipelines/Login/IdentifyCommerceUserProcessor.cs

```diff
@@ -0,0 +1,60 @@
+﻿//    Copyright 2020 EPAM Systems, Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+namespace Wooli.Foundation.Commerce.Infrastructure.Pipelines.Login
+{
+    using System;
+
+    using Account.Infrastructure.Pipelines.Login;
+
+    using Base.Infrastructure.Pipelines;
+    using Base.Models.Logging;
+    using Base.Services.Logging;
+
+    using Services.Tracking;
+
+    using Sitecore.Diagnostics;
+    using Sitecore.Pipelines;
+
+    public class IdentifyCommerceUserProcessor : PipelineProcessor<LoginPipelineArgs>
+    {
+        private readonly ICommerceTrackingService commerceTrackingService;
+
+        private readonly ILogService<CommonLog> logService;
+
+        public IdentifyCommerceUserProcessor(ICommerceTrackingService commerceTrackingService, ILogService<CommonLog> logService)
+        {
+            Assert.ArgumentNotNull(commerceTrackingService, nameof(commerceTrackingService));
+            Assert.ArgumentNotNull(logService, nameof(logService));
+
+            this.commerceTrackingService = commerceTrackingService;
+            this.logService = logService;
+        }
+
+        public override void Process(LoginPipelineArgs args)
+        {
+            Assert.ArgumentNotNull(args, nameof(args));
+
+            try
```

Comment:
Good, but need to move this try/catch + addMessage + errorLog to abstract `DoProcessSafe` and refactor usages.